lib/Cargo.toml

```diff
@@ -79,7 +79,7 @@ hex = { version = "0.4.3", optional = false }
 indexmap = { version = "2.1.0", features = ["serde"] }
 indxdb = { version = "0.4.0", optional = true }
 ipnet = "2.9.0"
-js = { version = "=0.4.0-beta.4", package = "rquickjs", features = ["array-buffer", "bindgen", "classes", "futures", "loader", "macro", "parallel", "properties","rust-alloc"], optional = true }
+js = { version = "=0.4.0", package = "rquickjs", features = ["array-buffer", "bindgen", "classes", "futures", "loader", "macro", "parallel", "properties","rust-alloc"], optional = true }
```

Comment:
Does the version still need to be pinned? Isn't `0.4.0` enough? Do you think `rquickjs ` might introduce breaking changes in `0.4.x`?

Reply:
Your right, it doesn't need to be pinned. I unpinned it in a new commit.